nebula_ros/include/nebula_ros/hesai/decoder_wrapper.hpp

```diff
@@ -64,6 +67,29 @@ class HesaiDecoderWrapper
       std::chrono::duration<double>(seconds));
   }
 
+  static custom_diagnostic_tasks::RateBoundStatus make_rate_bound_status(
```

Comment:
For these, is there a need to convert to relative for the parameters?
It may be more intuitive to read diagnostics that are 1:1 with the parameters, as per the camera rate diagnostics PR.

Reply:
The thought was that the absolute value is given by whatever RPM is set, and that the user has to calculate in their head (e.g. AT128: 200 RPM -> 10 Hz, QT128: 600 RPM -> 10 Hz, 1200 RPM -> 20 Hz, etc.).
Additionally, if it's absolute, the user now has to change 5 parameters instead of one when changing RPM. What do you think?

Reply:
It's unlikely that running at 20Hz as opposed to 10Hz would still follow the same linear pattern for bounds anyway (i.e, 10% margin would be 1Hz for 600rpm but 2Hz at 1200rpm - in reality I don't think higher frame rate = setting higher absolute margin).
Thinking about how this diagnostic would actually be used - I imagine the sensor would be setup, a rough value for max and min put in, and then adjusted depending one the actual operation in the vehicle - where the people doing the tuning would be unlikely to calculate based on RPM and more likely to look at the diagnostic result and then change the margin parameters to suit, based on the actual frame rate achieved.
@manato do you have an opinion on this?